src/tree/tree.scss

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$iui-active-outline: thin solid t(iui-color-foreground-primary);
+$iui-child-node-spacing: calc(36px * (var(--level, 0)));
+
+@mixin iui-tree {
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+}
+
+@mixin iui-tree-node {
+  @include iui-focus;
+  display: flex;
+  width: 100%;
+  align-items: center;
+  padding: $iui-xxs;
+  cursor: pointer;
+  box-sizing: border-box;
+  user-select: none;
+  margin-right: $iui-sm;
+  //min-height: $iui-component-height-small + (2 * $iui-xxs); //button height + padding
+  min-height: $iui-baseline * 3;
+
+  > .iui-icon {
+    @include iui-icons-default;
+    margin-right: $iui-s;
+    margin-left: $iui-sm + 1;
+  }
+
+  .iui-button + .iui-icon {
+    margin-left: $iui-xs + 1;
+  }
+
+  > .iui-button {
+    margin-right: $iui-s;
+    margin-left: $iui-s;
+
+    &:first-child {
+      margin-left: $iui-child-node-spacing;
+    }
+
+    > .iui-icon {
+      @media (prefers-reduced-motion: no-preference) {
+        transition: transform $iui-speed-fast ease-out;
+      }
+    }
+  }
+
+  > .iui-checkbox {
+    margin-left: $iui-s;
+    margin-right: $iui-child-node-spacing;
+  }
+
+  .iui-checkbox + .iui-node-label {
+    margin-left: $iui-s;
+  }
+
+  .iui-node-label {
+    min-width: 0;
+
+    &:only-child {
+      margin-left: $iui-child-node-spacing;
```

Comment:
These all margins look weird e.g. this one adds margin to the child but parent label also has margin.

Reply:
This one is needed to ensure the node is at the correct depth when the node label is the only element inside the node, like Node 2.2.1.1. In the nodes above the margin is added to the button but since this node doesn't have a button it needs to be added to the label instead.

![image](https://user-images.githubusercontent.com/88331924/143058312-753cdeb6-fc0f-4f54-a8d4-7fbbc4a7e98b.png)

Reply:
I added iui-node-content class to simplify these margins